docs/releasing/publishing-from-a-support-branch.md

```diff
@@ -0,0 +1,157 @@
+
+### Before you publish GOV.UK Frontend
+
+1. At stand up, person leading the release (usually a developer) to tell the GOV.UK Design System team we are close to releasing.
+
+2. Developers to raise new issues in the team GitHub repositories ([govuk-frontend](https://github.com/alphagov/govuk-frontend), [govuk-frontend-docs](https://github.com/alphagov/govuk-frontend-docs), [govuk-prototype-kit](https://github.com/alphagov/govuk-prototype-kit)) to:
+    - create announcement draft for the new release (example card: [#2108](https://github.com/alphagov/govuk-frontend/issues/2108))
+    - create release notes for the new release (example card: [#1986](https://github.com/alphagov/govuk-frontend/issues/1986))
+    - create release notes for the new release of GOV.UK Prototype Kit (example card: [#958](https://github.com/alphagov/govuk-prototype-kit/issues/958))
+    - create a card for the new release of GOV.UK Frontend (example card: [#1987](https://github.com/alphagov/govuk-frontend/issues/1987))
+    - update the GOV.UK Design System to use the new release of GOV.UK Frontend (example card: [#1347](https://github.com/alphagov/govuk-design-system/issues/1347))
+    - create a card for the new release of GOV.UK Prototype Kit (example card: [#917](https://github.com/alphagov/govuk-prototype-kit/issues/917))
+    - update the GOV.UK Prototype Kit to use the new release (example card: [#923](https://github.com/alphagov/govuk-prototype-kit/issues/923))
+
+3. Person leading the release to add the issues to the [Design System sprint board](https://github.com/orgs/alphagov/projects/4).
+
+4. Content designer to:
+    - write announcements to post on Slack after we release:
+      - GOV.UK Frontend (for example, [draft comms for the cookie banner component](https://docs.google.com/document/d/1jVyMB7i94NOeflWaf3kE4Q4APMXGfluK3rOh74IHO08/edit))
+      - GOV.UK Prototype kit
+    - check who the release’s contributors are and if we have consent to include their name
+
+5. Technical writer to finalise draft of release notes and get 2i on them.
+
+6. Content designer or designer to update community backlog with rationale for any decisions we made.
+
+7. Person leading the release to coordinate the sign-off when they are ready to do the release. Once the team agrees, this confirms a code and content freeze. Use the [#design-system-team-channel](https://gds.slack.com/app_redirect?channel=design-system-team-channel) to confirm sign-off from:
+    - content designer, technical writer and designers for guidance, examples and community backlog decision rationale
+    - technical writer and developers for Nunjucks macros
+    - developers for changes to GOV.UK Frontend
+    - technical writer for release notes
+    - content designer/Community manager/Technical writer for announcements and engagement activities
+
+Note: Before you go on leave, tell the delivery manager who will be looking after your work. This will help us to complete sign-off.
```

Comment:
```suggestion
Note: Before you go on annual leave, tell the delivery manager who will be looking after your work. This will help us to complete sign-off.
```